skimage/filters/tests/test_thresholding.py

```diff
@@ -1,3 +1,4 @@
+import pytest
```

Comment:
Sorry, I'm a bit late...
A while ago we decided not to use direct testing framework imports in the library, and to wrap those into `skimage._shared.testing`.

Reply:
@soupault sure, but as you know, this is not set in stone:

https://github.com/scikit-image/scikit-image/pull/3164#issuecomment-395649448

This might be a decision we revisit soon.

Reply:
@jni Hmm... Do you have in mind a better option? My thinking is that for that particular case (`skimage.test`) it is more convenient to use `pytest` directly because it allows one to run the tests for (1) the full package, (2) modules, (3) single tests, and (4) documentation using a similar syntax (also, the usage is already well-described in the guidelines - http://scikit-image.org/docs/stable/contribute.html#testing). The legacy solution only implemented (1) and (4), and had to be tested itself.